hyp3_rtc_gamma/rtc_sentinel.py

```diff
@@ -733,7 +747,7 @@ def rtc_sentinel_gamma(in_file,
     create_consolidated_log(out_name, lo_flag, dead_flag, match_flag, gamma_flag, roi,
                             shape, pwr_flag, filter_flag, pol, looks, log_file, smooth, terms,
                             no_cross_pol, par)
-    return 'PRODUCT'
+    return 'PRODUCT', out_name
```

Comment:
Both v1 and v2 entrypoints rename the PRODUCT dir to the product name.  I'd like to see that happen here and just return the one value, but I don't feel like digging through the v1 post-processing code to see if anything is hard-coded to look for PRODUCT/.

Reply:
I think we are making a clear improvement, might be good to push on and document these wants